python/lsst/ts/salobj/topics/read_topic.py

```diff
@@ -188,6 +190,10 @@ class ReadTopic(BaseTopic):
         If queue_len <= 0.
     ValueError
         If max_history > queue_len.
+        Note: the DDS write queue length sets an upper limit on how many
+        historical samples are available. However, dds offer no way
```

Comment:
I think comments need fixing, we can read the size of the queues thought python, no?

Reply:
Fixed